wolox-training/nh-android
# Training Android: Login FrontEnd

## Login FrontEnd

### Summary

Implemented the front end for the login.
Login view has fields to enter an email and a password, and two buttons for login and signup.
Implemented validation for the inputs when the login button is pressed.


**Cards:** 
* Card: [8] https://trello.com/c/EE6a56qt/8-login

### Test Plan
1) Login with no email and no password, there should be two errors saying that the fields are required, as shown in Screenshot 1
2) Login with invalid email like "mail" or "mail@test", there should be an error saying that the email is not valid, as shown in Screenshot 2
3) Login with a password shorter than 8 characters like "pass", there should be an error saying that the password should be at least 8 characters, as shown in Screenshot 3
4) Login with valid email like "mail@test.com" and valid password like "password", there should be no errors and no message, as shown in Screenshot 4

### Screenshots

#### Screenshot 1
![1](https://cloud.githubusercontent.com/assets/14595211/10143085/bf5f0a02-65ec-11e5-8f8e-b56871632ca1.png)

#### Screenshot 2
![2](https://cloud.githubusercontent.com/assets/14595211/10143087/c11c2136-65ec-11e5-9d71-27b9c787d13f.png)

#### Screenshot 3
![3](https://cloud.githubusercontent.com/assets/14595211/10143145/0c83db8c-65ed-11e5-8506-e3444ec71114.png)

#### Screenshot 4
![4](https://cloud.githubusercontent.com/assets/14595211/10143090/c38cbc28-65ec-11e5-905b-07b67f70b21e.png)
